bulk-user-export-tool/README.md

```diff
@@ -0,0 +1,49 @@
+# Bulk User Export Tool
+
+Java client that can export a bulk of users as a comma-separated values (.csv) using SCIM 2.0 APIs.
+
+## Table of contents
+
+- [Download and build](#download-and-build)
+- [Exporting user data from WSO2 IS](#exporting-user-data-from-wso2-is)
+
+## Download and build
+
+### Prerequisites
+
+* [Maven](https://maven.apache.org/download.cgi)
+* [Java](http://www.oracle.com/technetwork/java/javase/downloads)
+
+### Building from source
+
+1. Get a clone or download source of [WSO2 sample-is repository](https://github.com/wso2/samples-is).
+
+   We will refer this directory as `<IS_SAMPLE_REPO>` here onwards.
+2. Run the Maven command `mvn clean install` from the `<IS_SAMPLE_REPO>/bulk-user-export-tool/scim-bulk-user-export-tool` directory.
+
+You can find the tool in `target` directory of `<IS_SAMPLE_REPO>/bulk-user-export-tool/scim-bulk-user-export-tool`
+directory.
+Application distributions are named as `scim-bulk-user-export-tool-LATEST-jar-with-dependencies.jar`.
+
+## Exporting user data from WSO2 IS
+
+Follow these steps to export user data from WSO2 IS
+
+1. Start the WSO2 IS. (Note : This should SCIM 2.0 supported and enabled IS version(5.3.0 or higher))
+2. Build the tool from source by following steps in [Building from source](#building-from-source).
+3. Copy `scim-bulk-user-export-tool-LATEST-jar-with-dependencies.jar` inside `target` directory to scripts 
```

Comment:
At this step, point to the downloadable binary, and optionally mention if user build from the source then use the .jar in the target.